routing/bicycle_directions.cpp

```diff
@@ -210,20 +214,20 @@ Index::FeaturesLoaderGuard & BicycleDirectionsEngine::GetLoader(MwmSet::MwmId co
   return *m_loader;
 }
 
-void BicycleDirectionsEngine::LoadPathGeometry(FeatureID const & featureId,
+void BicycleDirectionsEngine::LoadPathGeometry(UniNodeId const & uniNodeId,
                                                vector<Junction> const & path,
                                                LoadedPathSegment & pathSegment)
 {
   pathSegment.Clear();
 
-  if (!featureId.IsValid())
+  if (!uniNodeId.GetFeature().IsValid())
```

Comment:
There's a ASSERT in GetFeature(). GetFeature could be called only for mwm type. Do you want me to add an extra ASSERT?